platformio.ini

```diff
@@ -38,6 +38,7 @@ lib_deps =
 [common_stm32]
 arduino_ver = >=4.10800
 ststm32_ver = >=6.1
+build_flags = -DTIM_IRQ_PRIO=13
```

Comment:
I anticipate moving more into this, for other commonalities across all STM32 environments. Some of the USB defines are good candidates.

I didn't do that now to limit the churn in this PR, and because some of the USB defines are going to change in the next framework release anyway. I'm told that release is anticipated within the next couple weeks. Hopefully it will be available through PlatformIO quickly so we will be able to rework the USB and SoftwareSerial configuration then.

Reply:
So these changes worked, as you already knew. Just thought I would confirm it on my end for you. 

Works as expected and sounds MUCH better